openslides/core/static/templates/index.html

```diff
@@ -15,92 +15,150 @@
+                  <div class="modal-body">
+                    <p ng-if='loginFailed' class="text-danger">
+                      <strong translate>Username or password is not correct.</strong>
+                    </p>
+                    <form>
+                      <div class="input-group form-group">
+                        <div class="input-group-addon"><i class="fa fa-user"></i></div>
+                        <input type="text" ng-model="username" class="form-control input-lg"
+                            placeholder="{{'Username'|translate}}">
+                      </div>
+                      <div class="input-group form-group">
+                        <div class="input-group-addon"><i class="fa fa-key"></i></div>
+                        <input type="password" ng-model="password" class="form-control input-lg"
+                            placeholder="{{'Password'|translate}}">
+                      </div>
+                      <div class="form-group">
+                        <button type="submit" ng-click="login(username, password)"
+                            class="btn btn-primary btn-lg btn-block" translate>
+                          Login
+                        </button>
+                      </div>
+                      <div class="form-group">
+                        <!-- TODO: if anonymous user is activate -->
+                        <button type="submit" class="btn btn-default" translate>
+                          Continue as guest
+                        </button>
+                      </div>
+                    </form>
+                  </div>
                 </div>
+              </div>
             </div>
+          </div>
         </div>
-    </nav>
-    <!-- Container -->
-    <div class="container-fluid" id="container">
-    <div class="row">
-
-        <!-- Sidebar navigation (main menu) -->
-        <div class="col-md-2 leftmenu lefticon">
-            <ul>
-                {% for entry in main_menu_entries %}
-                    <li{% if entry.is_active %} class="active"{% endif %}>
-                        <a href="{{ entry.get_url }}" class="tooltip-right">
-                        <!--TODO-->
-                        <span class="glyphicon {{ entry.get_icon_css_class }}" aria-hidden="true"></span>
-                        <span class="text">{{ entry }}</span>
-                        </a>
-                    </li>
-                {% endfor %}
-            </ul>
+        <!-- language switcher -->
+        <div class="btn-group" ng-controller="LanguageCtrl">
+          <button class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+            <i class="fa fa-flag"></i>
+            <span class="caret"></span>
+          </button>
+          <ul class="dropdown-menu dropdown-menu-right" role="menu">
+            <li><a href="" ng-click="switchLanguage('en')">
+                  <i class="fa fa-flag"></i>
+                  <translate>English</translate> (EN)
```

Comment:
The string '(EN)' is not translated. The same with DE and FR in line 114 and 118